src/DIRAC/WorkloadManagementSystem/Client/JobStatus.py

```diff
@@ -95,8 +95,8 @@ def __init__(self, state):
             RESCHEDULED: State(6, [WAITING, RECEIVED, DELETED, FAILED], defState=RESCHEDULED),
             MATCHED: State(5, [RUNNING, FAILED, RESCHEDULED, KILLED], defState=MATCHED),
             WAITING: State(4, [MATCHED, RESCHEDULED, DELETED], defState=WAITING),
-            STAGING: State(3, [WAITING, FAILED, KILLED], defState=STAGING),
-            SCOUTING: State(2, [CHECKING, FAILED, STALLED, KILLED], defState=SCOUTING),
+            STAGING: State(3, [CHECKING, WAITING, FAILED, KILLED], defState=STAGING),
```

Comment:
Does this state ID (3) have to be unique?

Reply:
No, but it better be.

Reply:
Ok, I will re-enumerate all states

Reply:
I have reassigned IDs to the states. Please check to confirm that you agree.